frontend/test/metabase/scenarios/collections/collections.cy.spec.js

```diff
@@ -161,80 +161,6 @@ describe("scenarios > collection_defaults", () => {
       });
     });
 
-    describe("managing items", () => {
-      it("should let a user move a collection item via modal", () => {
-        cy.visit("/collection/root");
-
-        // 1. Click on the ... menu
-        openEllipsisMenuFor("Orders");
-
-        // 2. Select "move this" from the popover
-        cy.findByText("Move this item").click();
-        modal().within(() => {
-          cy.findByText(`Move "Orders"?`);
-          // 3. Select a collection that has child collections and hit the right chevron to navigate there
-          cy.findByText("First collection")
-            .next() // right chevron icon
-            .click();
-          cy.findByText("Second collection").click();
-          // 4. Move that item
-          cy.findByText("Move").click();
-        });
-        // Assert that the item no longer exists in "Our collection"...
-        cy.findByText("Orders").should("not.exist");
-
-        openDropdownFor("First collection");
-        // ...and that it is indeed moved inside "Second collection"
-        cy.findByText("Second collection").click();
-        cy.findByText("Orders");
-      });
-
-      it("pinning an item workflow should work", () => {
-        cy.visit("/collection/root");
-        // Assert that we're starting from a scenario with no pins
-        cy.findByText("Pinned items").should("not.exist");
-
-        pinItem("Orders in a dashboard"); // dashboard
-        pinItem("Orders, Count"); // question
-
-        // Should see "pinned items" and items should be in that section
-        cy.findByText("Pinned items")
-          .parent()
-          .within(() => {
-            cy.findByText("Orders in a dashboard");
-            cy.findByText("Orders, Count");
-          });
-        // Consequently, "Everything else" should now also be visible
-        cy.findByText("Everything else");
-        // Only pinned dashboards should show up on the home page...
-        cy.visit("/");
-        cy.findByText("Orders in a dashboard");
-        cy.findByText("Orders, Count").should("not.exist");
-        // ...but not for the user without permissions to see the root collection
-        cy.signOut();
-        cy.signIn("none");
-        cy.visit("/");
-        cy.findByText("Orders in a dashboard").should("not.exist");
-      });
-
-      it.skip("should let a user select all items using checkbox (metabase#14705)", () => {
-        cy.visit("/collection/root");
-        cy.findByText("Orders")
-          .closest("a")
-          .within(() => {
-            cy.icon("table").trigger("mouseover");
-            cy.findByRole("checkbox")
-              .should("be.visible")
-              .click();
-          });
-
-        cy.findByText("1 item selected").should("be.visible");
-        cy.icon("dash").click();
-        cy.icon("dash").should("not.exist");
-        cy.findByText("4 items selected");
-      });
-    });
-
```

Comment:
@cam if it makes it easier for you - just accept "current change" here. This diff is correct - I moved this part to `permissions.cy.spec.js`